pkg/server/server_test.go

```diff
@@ -113,6 +129,13 @@ func TestMain(m *testing.M) {
 		logrus.Fatalf("failed to create json cache: %v", err)
 	}
 	s2 := server(fbcJsonStore)
+
+	fbcJsonDeprecationStore, err := fbcJsonCacheFromFs(validFS, filepath.Join(tmpDir, "json-deprecation-cache"))
+	if err != nil {
+		logrus.Fatalf("failed to create json deprecation cache: %v", err)
+	}
+	s3 := server(fbcJsonDeprecationStore)
```

Comment:
Similarly, this can be `deprecationServer` ... when I saw `s3.Serve()` I for a moment wondered if you were serving an S3-compatible object storage ... ?

Reply:
The predecessor objects are `s` and `s2`.  This holds to that pattern.

Reply:
I know. I am saying it's a bad pattern.